app/src/main/res/values/strings.xml

```diff
@@ -10,4 +17,6 @@
         <item>20 minutes</item>
         <item>30 minutes</item>
     </string-array>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
Is this being used anywhere as a placeholder?

Reply:
I'll remove this. I think android studio added it automatically.